composer.json

```diff
@@ -14,12 +14,13 @@
         }
     },
     "require": {
-        "php": "^5.5 || ^7.0"
+        "php": "^5.6 || ^7.0"
     },
     "require-dev": {
         "phpunit/PHPUnit": "~5.0 || ~6.0",
         "zendframework/zend-servicemanager": "^2.7.5 || ^3.0.3",
-        "zendframework/zend-coding-standard": "^1.0"
+        "zendframework/zend-coding-standard": "^1.0",
+        "doctrine/instantiator": "~1.0 || ^1.5"
```

Comment:
Why do we need `doctrine/instantiatior`?

Reply:
I'll remove this.  It was a dev issue I thought was an issue.